### ISSUE TITLE
No way to specify objects that have attributes and support iteration

### ISSUE THREAD

**Issue by thibaudcolas** [torchbox/django-pattern-library#10]
In GitLab by @bcdickinson on Dec 19, 2017, 22:15

Motivating example from: search results. `search_results` value is both iterated and accessed with dot notation:

```django
{% for result in search_results %}
```

```django
{% if search_results.paginator.count %}
```

The YAML format doesn't support specifying values that behave this way. A possible suggestion would be to allow numeric keys in the YAML e.g.

```yaml
search_results:
  paginator:
    count: 42
  0: ... # result 1
  1: ... # result 2
  ...
```

**Comment by thibaudcolas** [comment]
This seems like a fairly common scenario, that should at least be documented.

**Comment by thibaudcolas** [comment]
Mentioned in the docs as of #131.

**Comment by thibaudcolas** [comment]
Fixed in #141 :)
